app/src/main/java/com/nilhcem/blenamebadge/ui/badge_preview/CheckList.kt

```diff
@@ -1,5 +1,7 @@
 package com.nilhcem.blenamebadge.ui.badge_preview
 
-internal class CheckList {
-    var list = mutableListOf<Boolean>()
-}
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class CheckList(var list: MutableList<Boolean> = mutableListOf()) : Parcelable
```

Comment:
```suggestion
data class CheckList(var list: List<Boolean> = mutableListOf()) : Parcelable
```

Reply:
Sir, values are being added inside the checklist, List will not allow me to add values to it. Only mutable lists allow me to perform add operations

Reply:
OK, this is Kotlin List, not Java, so just write `var list = mutableListOf<Boolean>()`

Reply:
Ok sir

Reply:
@iamareebjamal Sir, It says 'A type annotation is required on a value parameter' when I change it to `var list = mutableListOf<Boolean>()`

Reply:
OK, let it be. My mistake, I did not realize it was Kotlin built in type and not java interface List